packages/neuron-ui/package.json

```diff
@@ -45,7 +45,9 @@
     "react-router-dom": "^4.3.1",
     "react-scripts": "2.1.5",
     "styled-components": "^4.1.3",
-    "typescript": "3.2.2"
+    "typescript": "3.2.2",
+    "qrcode.react": "0.9.3",
+    "@types/qrcode.react": "0.8.2"
```

Comment:
`@types/xx` should be listed in `devDependencies`

Reply:
ok